Earthfile

```diff
@@ -0,0 +1,16 @@
+VERSION 0.6
+
+image:
+    FROM DOCKERFILE -f Dockerfile .
+
+test:
+    FROM alpine
+    WORKDIR /test
+    RUN apk add go docker jq
+    ENV GOPATH=/go
+    COPY . .
+    WITH DOCKER \
+            --allow-privileged \
+            --load auroraboot:latest=+image
+        RUN go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo && /go/bin/ginkgo -r -p --randomize-all --procs 2 --fail-fast --flake-attempts 3 ./...
```

Comment:
if ginkgo is in teh go.mod file you can just use `go run` and it will pick the proper version!

Reply:
tried that, but didn't worked as we don't have it as we don't include ginkgo in the code, only in the tests :/